Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'sqlite3', '~> 1.3.6'
 
 # Capybara versions that support RSpec 3 only support RUBY_VERSION >= 1.9.3
 if RUBY_VERSION >= '1.9.3'
-  gem 'capybara', '~> 2.2.0', :require => false
+  gem 'capybara', '~> 2.2', :require => false
```

Comment:
I believe we need to be sure we don't break compatibility with the lowest version we currently support (out side of :bug: fixes / patches).